source/_integrations/unifiprotect.markdown

```diff
@@ -52,7 +52,7 @@ CKGP with Firmware v1.x **do NOT run UniFi OS**, you must upgrade to firmware v2
 
 The absolute **minimal** software version is `1.20.0` for UniFi Protect. If you have an older version, you will get errors trying to set up the integration. However, the general advice is the latest 2 minor versions of UniFi Protect and hardware supported by those are supported. Since UniFi Protect has its own release cycle, you should only upgrade UniFi Protect _after_ the next Home Assistant release comes out to ensure the new version is fully supported.
 
-Example: as of `2022.2.0` of Home Assistant, UniFi Protect `1.21.0-beta.2` is the newest UniFi Protect version. So the recommended versions of UniFi Protect to run for a `2022.2.0` version of Home Assistant are `1.20.0`, `1.20.1`, `1.20.2`, `1.20.3`, `1.21.0-beta.1`, or `1.21.0-beta.2`.
+Example: as of `2022.2.0` of Home Assistant, UniFi Protect `1.21.0-beta.2` is the newest UniFi Protect version. So the recommended versions of UniFi Protect to run for a `2022.2.0` version of Home Assistant are `1.20.0`, `1.20.1`, `1.20.2`, `1.20.3`, `1.21.0-beta.1`, `1.21.0-beta.2`, `1.21.0-beta.3`.
```

Comment:
I'm not sure if makes sense to keep updating these here since its going to change every few days.

Reply:
The plan is only keep updating until 2022.2 comes out. I have no intention of maintaining this, just need an accurate list of the versions for the release.

Reply:
I am _hoping_ 1.21.0 comes out before 2022.2.0 comes out so I can just remove all of the beta releases.